src/helpers/fetch/fetchFromUri.ts

```diff
@@ -44,6 +44,9 @@ export default async (options: FetchFromUriOptions) => {
     };
 
     const response = await axios.request(requestOptions);
+    if (response && response.headers && 'last-modified' in response.headers) {
```

Comment:
I believe this could be written shorter with [TS optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining):

```
 if (response?.headers && 'last-modified' in response.headers) {
// etc.
```